cmd/minishift/cmd/start.go

```diff
@@ -665,11 +670,19 @@ func initSubscriptionManagerFlags() *flag.FlagSet {
 // determineClusterUpParameters returns a map of flag names and values for the cluster up call.
 func determineClusterUpParameters(config *clusterup.ClusterUpConfig) map[string]string {
 	clusterUpParams := make(map[string]string)
+	valid, _ := openshiftVersion.IsGreaterOrEqualToBaseVersion(config.OpenShiftVersion, constants.RefactoredOcVersion)
+	if valid {
+		// Set default value for base config for 3.10
+		clusterUpParams["base-dir"] = baseDirectory
+	} else {
+		// This will only required to work with openshift < 3.10
+		for key, value := range dataDirectory {
+			clusterUpParams[key] = value
+		}
+	}
```

Comment:
So to support version < 3.10 we still using `dataDirectory`. Explains https://github.com/minishift/minishift/pull/2336/files#r195985445.